schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -80,7 +81,7 @@ dct:title
   a owl:Ontology ;
   vaem:hasGraphMetadata vaem:GMD_QUDT-SCHEMA ;
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;
-  rdfs:label "QUDT Schema - Version 2.1.1" ;
+  rdfs:label "QUDT Schema - Version 2.1.2" ;
```

Comment:
This would be more appropriate in the metadata object, yes? In there, vaem:title and vaem:revision are used during web page generation. To my knowledge, the rdfs:label in the ontology object is not used by anything.

Reply:
The rdfs:label is used in EDG